cim-forward-route/src/main/java/com/crossoverjie/cim/route/service/impl/AccountServiceRedisImpl.java

```diff
@@ -154,11 +155,12 @@ private void parseServerInfo(Map<Long, CIMServerResVO> routes, String key) {
 
     @Override
     public void pushMsg(CIMServerResVO cimServerResVO, long sendUserId, ChatReqVO groupReqVO) throws Exception {
-        CIMUserInfo cimUserInfo = userInfoCacheService.loadUserInfoByUserId(sendUserId);
+        Optional<CIMUserInfo> cimUserInfo = userInfoCacheService.loadUserInfoByUserId(sendUserId);
 
         String url = "http://" + cimServerResVO.getIp() + ":" + cimServerResVO.getHttpPort();
+        ServerApi serverApi = RpcProxyManager.create(ServerApi.class, okHttpClient);
+        SendMsgReqVO vo = new SendMsgReqVO(cimUserInfo.get().getUserName() + ":" + groupReqVO.getMsg(), groupReqVO.getUserId());
```

Comment:
It is not recommended to directly use the `java.util.Optional#get` method, as it defeats the purpose of using `Optional`. It is suggested that similar modifications be made in all relevant places.

```java
        cimUserInfo.ifPresent(userInfo -> {
            String url = "http://" + cimServerResVO.getIp() + ":" + cimServerResVO.getHttpPort();
            SendMsgReqVO vo = new SendMsgReqVO(userInfo.getUserName() + ":" + groupReqVO.getMsg(), groupReqVO.getUserId());
            serverApi.sendMsg(vo, url);
        });
```

Reply:
The previously incorrect usage of the non-standard java.util.Optional#get method has been revised upon our re-examination.